docs/README.md

```diff
@@ -1,74 +1,104 @@
+| `gallery`  | ![https://img.shields.io/badge/-Mandatory-red](https://img.shields.io/badge/-Mandatory-red) Displays the gallery with all the products found in the search.                                                                                                                                                                                               | `not-found` | Block containing a text and a description for the page that was not found in the search. It must be declared as a child of `search-not-found-layout`.  | 
+| `search-content`          | ![https://img.shields.io/badge/-Mandatory-red](https://img.shields.io/badge/-Mandatory-red) Decides, behind the scenes, which block will be displayed: either the `gallery` block (if products are found) or the `not-found` block (if the selected filters lead to an empty search results page). This means that both `gallery` and `not-found` must be declared as `search-content` children.                    |
+| `store.not-found#search`    | When configured, it displays a 404 error message whenever the server is not able to return what the browser request was or when it is configured to not handle that request.  |  
+| `search-fetch-more`          | Renders the `Fetch more` button if pagination is of type `show-more`. If it is set to infinite scroll, it shows the `Loader` when the bottom of the page is reached |
+| `search-fetch-previous`     | Renders the `Fetch previous` button if pagination is of type `show-more`. If it is set to infinite scroll, shows the `Loader` when the bottom of the page is reached. | 
+| `search-products-count-per-page` | Displays the total number of products being displayed in the search results page. | 
+| `order-by.v2`            | Allows users to choose the product ordination in the search results page.  | 
+| `filter-navigator.v3`        | Allows users to apply different filters to the search. On mobile, renders a button that, when clicked on, displays all available filters in a sidebar. | 
+| `total-products.v2`        | Displays the total amount of products found for that search. | 
+| `search-title.v2`         | Displays a title for the search that was done. |                                                                                              |
+
+:information_source: The Search Result app data may be displayed on two different pages of your store's theme: either **search page** (`store.search`) or on the **home page** (`store.home`). When added to the search page, the block that is used must be the `search-result-layout`, since it fetches data provided by the template's current search context. If you want to add the app to the home page, the block that must be used is the `search-result-layout.customQuery`. 
```

Comment:
```suggestion
:information_source: The Search Result app data may be displayed on **search pages** (`store.search`) or any other page. When added to the search page, the block that is used must be the `search-result-layout`, since it fetches data provided by the template's current search context. If you want to add the app to another page, the block that must be used is the `search-result-layout.customQuery`. 
```